bindings/java/iota-wallet-java/examples/src/main/Env.java

```diff
@@ -0,0 +1,11 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+public class Env {
+    public static final String NODE = "https://api.testnet.shimmer.network";
+    public static final String EXPLORER = "explorer.shimmer.network/testnet";
+    public static final String PASSWORD = "ENCRYPTION_PASSWORD";
```

Comment:
maybe rename to `STRONGHOLD_ENCRYPTION_PASSWORD` to make it more clear what kind of password this is